extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -310,8 +312,7 @@ export class RefreshManager {
     return new Promise(resolve => {
       this.refreshTimeoutId_ = this.timer_.delay(() => {
         this.state_ = RefreshLifecycleState.INITIAL;
-        this.getIntersectionObserverWithThreshold_(
-            this.config_.visiblePercentageMin).unobserve(this.element_);
+        this.unobserve();
```

Comment:
Why do we need to unobserve and then observe again afterall the element never changes as its the amp-ad element, right?  Wouldn't it be simpler to just leave the observe in place and then start the timeout when you first enter viewport after a refresh?  We can discuss outside of the PR